railties/lib/rails/application/configuration.rb

```diff
@@ -43,6 +43,7 @@ def initialize(*)
         @public_file_server                      = ActiveSupport::OrderedOptions.new
         @public_file_server.enabled              = true
         @public_file_server.index_name           = "index"
+        @assume_ssl                              = false
```

Comment:
This should get documented in https://github.com/rails/rails/blob/main/guides/source/configuring.md